### DISCUSSION TITLE
docker部署支持ARM架构吗？

### DISCUSSION THREAD

**Discussion by hzjsw** [zhayujie/chatgpt-on-wechat#1649] [Q&A]
我想在N1盒子OpenWRT上部署。盒子是晶晨S905系列的高阶芯片，四核 ARM A53架构，Mali-450 GPU。有同学说好像不支持ARM架构是真的吗？

**Comment by ZHAOZ1491** [comment]
> 我想在N1盒子OpenWRT上部署。盒子是晶晨S905系列的高阶芯片，四核 ARM A53架构，Mali-450 GPU。有同学说好像不支持ARM架构是真的吗？

可以，我也是s905盒子
